tests/scripts/kubeadm.sh

```diff
@@ -36,7 +36,7 @@ EOF
         sudo systemctl daemon-reload
     fi
 
-    sudo kubeadm init --skip-preflight-checks
+    sudo kubeadm init --skip-preflight-checks --kubernetes-version ${KUBE_VERSION}
```

Comment:
how did this work before if we were not passing the `--kubernetes-version` flag? was it always just taking the latest?

Reply:
Its works without passing the flag as well, without the flag it would always deploy the latest stable version. eg, if we want pass KUBE_VERSION=v1.6.1 and not pass the `---kubernetes-version`, the script would deploy kubelet, kubectl and kubeadm v1.6.1, but kubeadm would start latest stable v1.6 wich currently is 1.6.13.